src/components/Notifications/NoticeBoard.tsx

```diff
@@ -18,44 +19,44 @@ import Loading from "@/components/Common/Loading";
 import Page from "@/components/Common/Page";
 
 import routes from "@/Utils/request/api";
+import query from "@/Utils/request/query";
 import { formatName, relativeTime } from "@/Utils/utils";
 
-import { UserBareMinimum } from "../Users/models";
-import { useGetNotifications } from "./GetNotifications";
-import { NotificationData } from "./models";
+import { NoticeData } from "./models";
 
 export const NoticeBoard = () => {
   const { t } = useTranslation();
-  const { data, loading } = useGetNotifications(routes.getNotifications, {
-    query: { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
+  const { data, isLoading } = useQuery({
+    queryKey: [
+      routes.getNotifications.path,
+      { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
+    ],
+    queryFn: query(routes.getNotifications, {
+      queryParams: { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
+    }),
+    enabled: true,
+    refetchOnWindowFocus: false,
   });
-
-  const [selectedNotice, setSelectedNotice] = useState<NotificationData | null>(
-    null,
-  );
-
-  interface UserInfoProps {
-    user: UserBareMinimum;
-    createdDate: string;
-  }
   console.log(data?.results);
-  const UserInfo: React.FC<UserInfoProps> = ({ user, createdDate }) => (
-    <div className=" py-2 flex items-center bg-gray-200 rounded-lg">
+  const [selectedNotice, setSelectedNotice] = useState<NoticeData | null>(null);
+
+  const UserInfo: React.FC<NoticeData> = (notice) => (
+    <div className="flex items-center bg-gray-200 rounded-lg">
       <Avatar
-        name={user.username}
-        imageUrl={user.read_profile_picture_url}
-        aria-label={`${formatName(user)}'s avatar`}
+        name={notice.caused_by.username}
```

Comment:
you should be passing the name of the user and not the username

```suggestion
        name={formatName(notice.caused_by)}
```